[DOCS][PT FE] Update pytorch conversion docs (#19396)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

                 PyTorch format to the OpenVINO Intermediate Representation. 

This page provides instructions on how to convert a model from the PyTorch format to the OpenVINO IR format.

The conversion is a required step to run inference using OpenVINO API.
It is not required if you choose to work with OpenVINO under the PyTorch framework, 
using its :doc:`torch.compile feature <pytorch_2_0_torch_compile>`.

Converting a PyTorch model with PyTorch Frontend 
###############################################################

To convert a PyTorch model to the OpenVINO IR format, use the OVC API (superseding the previously used tool, MO). To do so, use the ``convert_model()`` method, like so:

.. code-block:: py
   :force:

   import torchvision
   import torch

>    from openvino.tools.ovc import convert_model

Author: ```suggestion
   from openvino.tools.mo import convert_model
```

> This page provides instructions on how to convert a model from the PyTorch format to the OpenVINO IR format using Model Optimizer.
> Model Optimizer Python API allows the conversion of PyTorch models using the ``convert_model()`` method.

Reviewer: ```suggestion
This page provides instructions on how to convert a model from the PyTorch format to the OpenVINO IR format.
Even though you can run inference of PyTorch models directly in OpenVINO, you may consider this approach to
perform additional model optimizations and minimize first-inference latency, especially important for 
setups with large numbers of consecutive inference jobs.

```

Converting a PyTorch model with PyTorch Frontend 
###############################################################

> Example of PyTorch model conversion:

Reviewer: ```suggestion
To convert a PyTorch model explicitly to the OpenVINO IR format, use the OVC API (superceding the previously used tool, MO).
It is done with just one method, ``convert_model()``, like so:

```

* ``torch.jit.ScriptModule``
* ``torch.jit.ScriptFunction``

> Converting certain PyTorch models may require model tracing, which needs ``example_input`` parameter to be set.

Reviewer: ```suggestion
Converting certain PyTorch models may require model tracing, which needs the ``example_input`` 
parameter to be set, for example:
```

> 
> Example of using ``example_input``:

Reviewer: ```suggestion
```

* ``torch.jit.ScriptModule``
* ``torch.jit.ScriptFunction``

Converting certain PyTorch models may require model tracing, which needs the ``example_input`` 
parameter to be set, for example:

.. code-block:: py
   :force:

   import torchvision
   import torch

>    from openvino.tools.ovc import convert_model

Author: ```suggestion
   from openvino.tools.mo import convert_model
```

* ``list`` or ``tuple`` with tensors (``openvino.runtime.Tensor`` / ``torch.Tensor`` / ``np.ndarray``)
* ``dictionary`` where key is the input name, value is the tensor (``openvino.runtime.Tensor`` / ``torch.Tensor`` / ``np.ndarray``)

> Sometimes ``convert_model`` will produce inputs of the model with dynamic rank or dynamic type, such model may not be supported by hardware plugin,
> to solve this, please use ``input`` argument of ``convert_model``, please refer to `Convert Models Represented as Python Objects <openvino_docs_MO_DG_Python_API>` for more information. 

Reviewer: ```suggestion
Sometimes ``convert_model`` will produce inputs of the model with dynamic rank or dynamic type. 
Such model may not be supported by the hardware chosen for inference. To avoid this issue,
use the ``input`` argument of ``convert_model``. For more information, refer to `Convert Models Represented as Python Objects <openvino_docs_MO_DG_Python_API>`. 
```

.. important::

   The ``convert_model()`` method returns ``ov.Model`` that you can optimize, compile, or save to a file for subsequent use.

Exporting a PyTorch Model to ONNX Format
########################################

> It is also possible to export PyTorch model to ONNX and then converting it to IR. To convert and deploy a PyTorch model, follow these steps:

Reviewer: ```suggestion
It is also possible to export a PyTorch model to ONNX and then convert it to OpenVINO IR. To convert and deploy a PyTorch model this way, follow these steps:
```
